### PR TITLE
include nc 13 in travis test and adjust cases accordingly

### PR DESCRIPTION
**Pull request by blizzz**
to solve #425 

let's see whether i caught everything

### CODE REVIEW

File: templates/js_tpl.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
                 <&= escapeHTML(title == '' ? encodeURI(url) : title ) &>
             </a>
             <span class="bookmark_edit bookmark_edit_btn">
-                <img class="svg" src="<?php print_unescaped(OCP\image_path("", "actions/rename.svg")); ?>" title="<?php p($l->t('Edit')); ?>">
```
Comment (marcelklehr) [inline templates/js_tpl.php]: (see #426 ;))

**Comment by blizzz** [comment]
fixed with recent commit :)

File: templates/js_tpl.php [hunk 2]
```diff
@@ -51,11 +51,11 @@
     <li><a href="" class="tag"><&= escapeHTML(tag) &></a>
         <div class="tags_actions">
             <span class="tag_delete">
-                <img class="svg" src="<?php print_unescaped(OCP\image_path("", "actions/delete.svg")); ?>"
+                <img class="svg" src="<?php print_unescaped(image_path("", "actions/delete.svg")); ?>"
                      title="<?php p($l->t('Delete')); ?>">
             </span>
             <span class="tag_edit">
-                <img class="svg" src="<?php print_unescaped(OCP\image_path("", "actions/rename.svg")); ?>"
```
Comment (marcelklehr) [inline templates/js_tpl.php]: Is this a good idea? I have no clue how the templating works, though. From this I assume, it's a separate scope, which would give this a *lgtm* verdict.

**Review by marcelklehr — APPROVED** [review]
lgtm :ship:

File: templates/js_tpl.php (hunk 2 above)
Comment (blizzz) [reply templates/js_tpl.php]: yeah, he methods are defined in ``lib/private/legacy/template/functions.php`` and also called by the Template class. These short forms are also used in server and shipped apps.

**Pull request merged**